### PR TITLE
Api validation 186141023

### PR DESCRIPTION
**Pull request by hannahb-cogapp**

### CODE REVIEW

File: app/Http/Controllers/API/CustomTourController.php [hunk 1]
```diff
@@ -10,9 +10,22 @@ class CustomTourController extends BaseController
 {
     public function store(Request $request)
     {
-        $data = $request->json()->all();
-
-        $record = CustomTour::create(['tour_json' => json_encode($data)]);
+        $data = $request->validate(
+            [
```
Comment (tristanr-cogapp) [inline app/Http/Controllers/API/CustomTourController.php]: Move these to `app/Http/Requests/API/CustomTourRequest`

File: app/Http/Controllers/API/CustomTourController.php [hunk 2]
```diff
@@ -28,4 +41,20 @@ public function show(Request $request, $id)
         $tour_json = json_decode($custom_tour->tour_json, true);
         return response()->json(['tour_json' => $tour_json], 200);
     }
+
+    private function sanitizeData($data)
```
Comment (tristanr-cogapp) [inline app/Http/Controllers/API/CustomTourController.php:43]: Please add a doc comment

**Review by tristanr-cogapp — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/API/CustomTourController.php (hunk 1 above)
Comment (hannahb-cogapp) [reply app/Http/Controllers/API/CustomTourController.php]: Thanks, I've now moved the validation rules to `app/Http/Requests/API/CustomTourRequest`

File: app/Http/Controllers/API/CustomTourController.php (hunk 2 above)
Comment (hannahb-cogapp) [reply app/Http/Controllers/API/CustomTourController.php:43]: I've also added a doc comment explaining what this method does

**Review by tristanr-cogapp — APPROVED** [review]

**Pull request merged**